File: programmers/python/입문100/25.3_연속된수의합/25-3.py
def solution(k, total):
    answer = []
    # 시작 숫자가 n이고, 개수가 k 일때
    # n*k + ((k-1)*k)/2 = total 이다
    # k와 total을 알면, n = (total - (((k-1)*k)/2))/k
    n = (total - (((k-1)*k)/2))/k
    for i in range(k): # k개 넣기
        answer.append(n+i)
    return answer

File: programmers/java/기초124/25.3_정사각형으로만들기/25_3.java
class Solution {
    public int solution(int[][] arr) {
        // int answer = 0;
        // return answer;
        int n = arr.length;
        for (int i = 0l i < n; i++) {
            for (int j = i + 1; j < n ; j++) {
                if (arr[i][j] != arr[j][i]) {
                    return 0;
                }
            }
        }
        return 1;
    }
}